insights_client/__init__.py

```diff
@@ -28,26 +27,62 @@ def log(msg):
     print(msg, file=sys.stderr)
 
 
+def egg_version(egg):
+    '''
+    Determine the egg version
+    '''
+    if not sys.executable:
+        return None
+    try:
+        proc = Popen([sys.executable, '-c', 'from insights.client import InsightsClient; print(InsightsClient(None, False).version())'],
+                     env={'PYTHONPATH': egg, 'PATH': os.getenv('PATH')}, stdout=PIPE, stderr=PIPE)
+    except OSError as e:
+        logger.root.error(e)
+        return None
+    stdout, stderr = proc.communicate()
+    return stdout
+
+
+def sorted_eggs(eggs):
+    '''
+    Sort eggs to go into sys.path by highest version
+    '''
+    if len(eggs) < 2:
+        # nothing to sort
+        return eggs
+    if egg_version(eggs[0]) > egg_version(eggs[1]):
```

Comment:
Perhaps you should use a version parsing library to do the compare?  e.g. `packaging.version` or `distutils.version.LooseVersion` (if `packaging.version` isn't in rhel 6)

Reply:
Will investigate